src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball(colorSupplier.getRandomColor(), random.nextInt(100));
+        return ball;
```

Comment:
```suggestion
        return  new Ball(colorSupplier.getRandomColor(), random.nextInt(MAX_NUMBER));
```

[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery): Don’t create temporary variables if you can directly return value from the method
And lets make number 100 as constant field